.github/workflows/tests.yaml

```diff
@@ -1,236 +1,221 @@
 name: tests
-'on':
-  pull_request:
-    paths-ignore:
-      - '**.md'
-      - .github/ISSUE_TEMPLATE/**
-  push:
-    paths-ignore:
-      - '**.md'
-      - .github/ISSUE_TEMPLATE/**
-    branches:
-      - main
-      - pr/**/ci
+"on":
+    pull_request:
+        paths-ignore:
+            - "**.md"
+            - .github/ISSUE_TEMPLATE/**
+    push:
+        paths-ignore:
+            - "**.md"
+            - .github/ISSUE_TEMPLATE/**
+        branches:
+            - main
+            - pr/**/ci
 concurrency:
-  group: '${{ github.workflow }}-${{ github.head_ref || github.ref || github.run_id }}'
-  cancel-in-progress: true
+    group: "${{ github.workflow }}-${{ github.head_ref || github.ref || github.run_id }}"
+    cancel-in-progress: true
 env:
-  CARGO_TERM_COLOR: always
+    CARGO_TERM_COLOR: always
 jobs:
-  check:
-    name: Check
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-      - uses: actions-rs/cargo@v1
-        with:
-          command: check
-  test:
-    name: Test
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
```

Comment:
This pull request removes [`actions-rs/cargo@v1`](https://github.com/actions-rs/cargo/tree/v1/) and [`actions-rs/toolchain@v1`](https://github.com/actions-rs/toolchain/tree/v1/), which have both been archived for almost a year.